core/src/main/java/com/adobe/aem/commons/assetshare/search/results/Results.java

```diff
@@ -54,7 +54,14 @@
     long getTotal();
 
     /**
-     * @return true if there are more results. false if there are no more results.
+     * @return value of has more from search results. This method that not takes into account that guessTotal is bigger then all results
+     */
+    default boolean isHasMoreSearchResult(){
```

Comment:
Ive been staring at this method name for 10 mins now and it's not sitting right.

WDYT about naming it "hasMoreThanTotal()" ?  .. its sort of weird, but i think its the most accurate name (and slightly sets it apart from isMore()) (which i hate that we named it that .... hasMore... HASMORE!!! :))